### PR TITLE
Range input

### PR DESCRIPTION
**Pull request by ivan-aksamentov**
## Related issues and PRs

 - 

## Description

This is an experimental range input from @gj262.

### CODE REVIEW

**Comment by gj262** [comment]
thanks for opening the PR @ivan-aksamentov. I think it is ready to check out in this state. 

Readiness for merge is another matter. There are still issues w.r.t.:

- long fractional numbers from scenarios.json. 
- the component does not check that in the tuple [a, b] a < b. I was hoping to do that with yup. then all error handling can be done in the Formik aware FormRangeInput. But yup dependency checks are hard.

File: src/components/Form/RangeInput.tsx [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+  value?: RangeInputValue
+  onChange?: (value: RangeInputValue) => void
+  title?: string
+  placeholder?: string
+  hasError?: boolean
+  separator?: string
+  hint?: string
+}
+
+export default function RangeInput({
```
[the remark this answers is not in the document]
Comment (gj262) [reply src/components/Form/RangeInput.tsx:18]: All the way to 11! 🤘

**Comment by ivan-aksamentov** [comment]
@gj262 Thanks for exploring this. We evaluated different widgets and decided to go with #604 for now.

**Comment by gj262** [comment]
Sounds good. I agree with your comment "Implementing from scratch an input
that does not suck is a non-trivial task and there are many examples of
failure on npm".

**Pull request closed without merging**